packages/cdk/lib/cloudwatch-logs-management.ts

```diff
@@ -18,6 +18,7 @@ export interface CloudwatchLogsManagementProps
 	extends Omit<GuStackProps, 'stage' | 'env'> {
 	retentionInDays?: number;
 	logShippingPrefixes?: string[];
+	containsPIIData?: boolean;
```

Comment:
minor: With this type, we have three possible values: `undefined`, `true`, and `false`. We usually treat `undefined` and `false` as the same. I think this is the case here? In which case I'd suggest a strict boolean, with a default value, would be easier to reason about.

Reply:
By this, do you mean I should have the containsPIIData field be non-optional?
I'm not sure how exactly to do this without then requiring every stack to specify containsPIIData.
Currently the field is [defaulted to false](https://github.com/guardian/cloudwatch-logs-management/blob/609a49c9ca9e71730689787c05b4b0c2a45378c0/packages/cdk/lib/cloudwatch-logs-management.ts#L30) in the cloudwatch logs management constructer, where the other interface fields also receive a default value. 
The only other approach I can think of to do this would be to have an intermediate interface which applies the defaults and then use this in the constructor. Happy to do this if it is preferable.

Reply:
Ah, you're 100% correct. Apologies, I misinterpreted the code.

Reply:
Confirmed with Akash that the current default setting is the recommended approach.